src/main/java/uk/gov/digital/ho/hocs/queue/common/MessageType.java

```diff
@@ -1,12 +1,12 @@
 package uk.gov.digital.ho.hocs.queue.common;
 
-public enum MessageTypes {
+public enum MessageType {
     UKVI_COMPLAINTS("UKVI_COMPLAINTS"),
```

Comment:
[OOS] - we need to look at the naming of these. 

Likely wants to be [COMP_WEBFORM, CMS_MIGRATION]